src/applications/gqrx/receiver.cpp

```diff
@@ -853,13 +873,40 @@ receiver::status receiver::set_agc_decay(int decay_ms)
     return STATUS_OK; // FIXME
 }
 
-/** Set fixed gain used when AGC is OFF. */
-receiver::status receiver::set_agc_manual_gain(int gain)
+/** Get AGC current gain. */
+float receiver::get_agc_gain()
 {
     if (rx->has_agc())
-        rx->set_agc_manual_gain(gain);
+        return rx->get_agc_gain();
+    else
+        return 0;
+}
 
-    return STATUS_OK; // FIXME
+/** Set audio mute. */
+receiver::status receiver::set_mute(bool mute)
+{
+    if (d_mute == mute)
+        return STATUS_OK;
+    tb->lock();
+    if (mute)
```

Comment:
This tb connect/disconnect logic needs to be wrapped with a check making sure `d_demod != RX_DEMOD_OFF`. Otherwise, the application will crash if you try to mute it while the demodulator is off.

Reply:
Thanks.
It looks, like I have lost some changes during rebase. I'll fix this and update the PR.

Reply:
Updated.